emission/tests/storageTests/TestTokenQueries.py

```diff
@@ -10,12 +10,15 @@
 import emission.storage.decorations.token_queries as esdt
 import emission.net.auth.token_list as enat
 
-import bin.auth.insert_tokens as bai
 
 class TestTokenQueries(unittest.TestCase):
-    def setUp(self):
-        self.testUserId = uuid.uuid3(uuid.NAMESPACE_URL, "mailto:test@test.me")
-        edb.get_token_db().delete_many({})
+
+    def tearDown(self):
+    #All tests insert tokens of length one. Delete them once the test is done.
+    #Shouldn't affect "real" tokens with greater length that might be in the DB
+        for t in edb.get_token_db().find({}):
+            if len(t["token"]) == 1:
+                edb.get_token_db().delete_one(t)
```

Comment:
I am merging this right now, but would like to see some description of how you tested that this works.